### PR TITLE
[Flink] Fix unstable RemoteLookupJoinITCase.testServiceFileCleaned test method.

### LINKED ISSUE #3641: [Bug] RemoteLookupJoinITCase.testServiceFileCleaned is unstable
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/paimon/issues) and found nothing similar.

### Paimon version

0.9

### Compute Engine

flink

### Minimal reproduce step

https://github.com/apache/paimon/actions/runs/9729518905/job/26851511142
https://github.com/apache/paimon/actions/runs/9729521981/job/26851517274

### What doesn't meet your expectations?

enable it.

### Anything else?

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by zhuangchong**
### Purpose

Linked issue: close #3641

### CODE REVIEW

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java [hunk 1]
```diff
@@ -109,4 +111,9 @@ public InternalRowSerializer createValueSerializer() {
     public void close() throws IOException {
         client.shutdown();
```
Comment (JingsongLi) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java:112]: `networkClient.shutdown().get(10L, TimeUnit.SECONDS);`

10 s is not enough?

Comment (zhuangchong) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java:112]: The two CI logs mentioned in issue #3641 show that the running time is greater than 10s.

```
https://github.com/apache/paimon/actions/runs/9729518905/job/26851511142
https://github.com/apache/paimon/actions/runs/9729521981/job/26851517274
```

image

Comment (JingsongLi) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java:112]: can we just modify this to 60 s?

Comment (JingsongLi) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java:112]: keep your changes and modify this to 60 s.

Comment (zhuangchong) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java:112]: Done.

**Comment by JingsongLi** [comment]
+1

**Pull request merged**